### ISSUE TITLE
Exception when double-clicking Song 

### ISSUE THREAD

**Issue by bembleton** [BleuBleu/FamiStudio#94]
I got a `DivideByZeroException` while editing BPM and notes per beat and pattern for SFX. It's not fatal, but I'm unable to edit the song properties again once it's in this state. The exact BPM selected might not matter.

```
System.DivideByZeroException: Attempted to divide by zero.
   at FamiStudio.FamiStudioTempoUtils.ComputeBpmForGroove(Boolean pal, Int32[] groove, Int32 notesPerBeat) in C:\Users\bleub\Documents\GitHub\FamiStudio\FamiStudio\Source\Utils\TempoUtils.cs:line 110
```

## Steps to Reproduce
1. Create a new Song
2. Double-click Song name and set
    - Song Length: 1
    - Notes per Beat: 1
    - Notes per Pattern: 1
    - BPM: 360
2. Double-click Song name again and set
    - BPM: 300
    - Leave notes exactly where they are
3. Double-click Song name again

## Environments
famistudio : 3.1.1

**Comment by BleuBleu** [comment]
This is likely the same issue as https://github.com/BleuBleu/FamiStudio/issues/90.

If that's the case, then i already have a fix for 3.2.0, otherwise will fix too! 

Thanks a lot for the bug report.

-Mat

**Comment by bembleton** [comment]
Nice. I saw that other issue but the call stack and exception looked pretty different so I made another just in case. Cheers!

**Comment by BleuBleu** [comment]
Fixed in 3.2.0.
